lists.js

```diff
@@ -2,7 +2,7 @@
 
     lists.js
 
-    list data structure and GUI for SNAP!
+    list data structure and GUI for Snap!
```

Comment:
same here, let's not just change the uppercase/lowercase spelling of SNAP for every file...